scripts/release-region.sh

```
@@ -0,0 +1,113 @@
+# Skip everything except build+archive - we just want the artifacts
+release:
+  disable: true
+EOF
+
+goreleaser release --config .goreleaser.region.yaml --skip=validate,announce,publish
+
+# Step 3: Clean up to free disk space for next region
+echo "Cleaning up build cache..."
+rm -f .goreleaser.region.yaml
+# Remove the raw binaries but keep archives
+rm -rf dist/${REGION}_*
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)